xla/stream_executor/cuda/cuda_driver.cc

```diff
@@ -1221,75 +1253,75 @@ absl::Status GpuDriver::LoadPtx(GpuContext* context, const char* ptx_contents,
                                 CUmodule* module) {
   absl::Notification notification;
   absl::Status ret = absl::OkStatus();
-  GetDriverExecutor()->Schedule([context, ptx_contents, module, &ret,
-                                 &notification]() {
-    ScopedActivateContext activation(context);
-    void* ptx_data = const_cast<char*>(ptx_contents);
-    static const unsigned int kLogBufferBytesLimit = 1024;
-    unsigned int error_log_buffer_bytes = kLogBufferBytesLimit;
-    unsigned int info_log_buffer_bytes = kLogBufferBytesLimit;
-    absl::InlinedVector<char, 4> error_log_buffer(error_log_buffer_bytes);
-    absl::InlinedVector<char, 4> info_log_buffer(info_log_buffer_bytes);
-    bool log_verbose = true;
-    CUjit_option options[] = {CU_JIT_ERROR_LOG_BUFFER_SIZE_BYTES,
-                              CU_JIT_ERROR_LOG_BUFFER,
-                              CU_JIT_INFO_LOG_BUFFER_SIZE_BYTES,
-                              CU_JIT_INFO_LOG_BUFFER, CU_JIT_LOG_VERBOSE};
-    // Note that the driver API wants the contents of this values to be stored
-    // in an array of void*s, so we coerce them accordingly.
-    void* option_values[] = {
-        absl::bit_cast<void*>(uintptr_t(error_log_buffer_bytes)),
-        absl::bit_cast<void*>(error_log_buffer.data()),
-        absl::bit_cast<void*>(uintptr_t(info_log_buffer_bytes)),
-        absl::bit_cast<void*>(info_log_buffer.data()),
-        absl::bit_cast<void*>(uintptr_t(log_verbose))};
-    CHECK(TF_ARRAYSIZE(options) == TF_ARRAYSIZE(option_values));
-
-    absl::Status status;
-    {
-      // TODO(leary) Need to see if NVIDIA can expunge the leakiness in their
-      // module loading: see http://b/13248943
-      absl::LeakCheckDisabler disabler;
-      status = cuda::ToStatus(cuModuleLoadDataEx(
-          module, ptx_data, TF_ARRAYSIZE(options), options, option_values));
-    }
+  GetDriverExecutor()->Schedule(
```

Comment:
This looks just a formatting change? Or I can't find a real one? If it's just formatting can you please revert it, I'll submit NFC formatting later separately

Reply:
Revert the change here, should be due to clicking the whole file format button from vscode.